samples/Mvc.RenderViewToString/RazorViewToStringRenderer.cs

```diff
@@ -34,15 +36,7 @@ public class RazorViewToStringRenderer
         public async Task<string> RenderViewToStringAsync<TModel>(string name, TModel model)
```

Comment:
Sure. My concern was mostly about having to specify the `.cshtml` part of the path.